src/test/resources/sample/external/rest/ext1.external.headers

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't see any usages of this file?

Reply:
It is used ImporterTest.testImportExternalResource()

Reply:
Okay, I dont' understand the mechanism causing it to be pulled in, but it does appear to be necessary